### DISCUSSION TITLE
Need clarity on 1.4.1 Use of Color and 1.4.11 Non-text Contrast Success criteria's

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#3609] [Q&A]
Hi Team,

I'd like to bring to your attention a scenario regarding the color contrast and focus indicator for the 'Co-browse a web page' button.

In default state of 'Co-browse a web page' button, the background color is set to white (#FFFFFF), and upon keyboard focus, it changes to grey (#E5E5E5). The contrast ratio between these two colors is 1.3:1, which is below the 3:1 contrast ratio so would be failing 1.4.1 Use of Color Success criteria here?

Additionally, considering that the grey color is serving as a focus indicator, are we also evaluating the focus indicator against the default white background,
and would be failing 1.4.11 Non-text Contrast Success criteria too?

![image](https://github.com/w3c/wcag/assets/115781693/c1215953-d34c-4821-9441-8ce2777c4869)

**Comment by JAWS-test** [comment]
If you look at the [C15 technique](https://www.w3.org/WAI/WCAG22/Techniques/css/C15), you might get the impression that a focus indicator that does not have sufficient contrast should be failed at 1.4.1 or 2.4.7. However, I suspect that the technique was written when WCAG 2.1 or 2.2 did not yet exist. Since WCAG 2.1, it makes more sense to evaluate a focus indicator with insufficient contrast to the background at 1.4.11. Since WCAG 2.2, insufficient contrast between the focus and no focus states is only an AAA-SC (2.4.13) and therefore may not actually be rated at level AA (not even at 1.4.1 or 2.4.7). However, the contrast of the focus indicator to the background can still be checked at 1.4.11.

Conclusion: I think, the example from your screenshots is therefore a violation of 1.4.11

**Reply by AccessibilityEnthusiastic** [reply]
@JAWS-test I checked the Figure 15 
https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html

As per this:
If the focus state relies on a change of color (e.g., changing only the background color of a button),
then focused and unfocused states of the background color comparison also needs to be done for the Use of color.

![image](https://github.com/w3c/wcag/assets/115781693/1a0751ad-7cbb-4fa2-bf45-ba258ad94f7d)

**Reply by JAWS-test** [reply]
@AccessibilityEnthusiastic You're right, I had overlooked that. I don't think it's been in Understanding for that long either. However, I wonder why we need the new 3:1 requirement for the focus indicator in 2.4.13 if this has always been checked by 1.4.1.

**Comment by mraccess77** [comment]
The example in figure 15 doesn't use color (hue) it relies on lightness to distinguish the buttons. This is an area where we need more clarity because typically the group has said that the comparison between focused and non-focused state is not covered by the SC and that 1.4.1 is about hue.

**Reply by patrickhlauke** [reply]
the "is it *just* a change of lightness/contrast or also of hue" aspect is the trickiest to really normatively pin down. FWIW the hue (of the light blue and dark blue) is slightly/fractionally different (in `hsl` terms, `hsl(208, 42%, 53%)` vs `hsl(215, 59%, 39%)` so the `h` value deviates). but when i recreated this image (in https://github.com/w3c/wcag/pull/2574) i made sure that at least the difference between the two shades of blue was less than a `3:1` ratio, so it *does* fail use of color
